app/src/main/java/org/p2p/wallet/auth/repository/RestoreFlowDataLocalRepository.kt

```diff
@@ -28,6 +30,7 @@ class RestoreFlowDataLocalRepository {
     var userPhoneNumber: PhoneNumber? = null
         set(value) {
             field = value
+            isRestoreWalletRequestSent = true
```

Comment:
you're setting it to true using interactor, don't implicitly set it to true here